### PR TITLE
Use Rabbitmq instead of Redis for task queues

### PR DESCRIPTION
**Pull request by Uxio0**
## Background
Redis is not working pretty well when we have a lot of tasks

## Solution
- Use RabbitMQ as the recommended queue manager
- Priority numbers are reversed when using Redis or RabbitMQ (take a look at https://docs.celeryq.dev/en/stable/userguide/routing.html -> Priority Order)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,6 +23,11 @@ services:
       - --save ""
       - --appendonly no
 
+  rabbitmq:
+    image: rabbitmq:alpine
+    # ports:
+    #  - "5555:5555"
```
Comment (fmrsabino) [inline docker-compose.yml:29]: Shall we specify the ports here?

**Review by fmrsabino — APPROVED** [review]
🐇

File: docker-compose.yml (hunk 1 above)
Comment (Uxio0) [reply docker-compose.yml:29]: No, as I don't think we should expose them

Comment (Uxio0) [reply docker-compose.yml]: Removed

**Pull request merged**